js/binance.js

```diff
@@ -2931,7 +2932,7 @@ module.exports = class binance extends Exchange {
             method = 'fapiPrivatePostOrder';
         } else if (marketType === 'delivery') {
             method = 'dapiPrivatePostOrder';
-        } else if (marketType === 'margin') {
```

Comment:
Since all margin markets are spot markets, couldn't we accept this configuration?

```
type = 'spot' and params = { 'margin' : true}
```

Reply:
we don't know if this is cross or isolated

Reply:
@samgermain yes but we're also accepting `'marketType'= 'margin'` and reading the defaultValue of `marginMode` so I don't see why accepting `{ 'margin' : true}` is different 🙂

Reply:
@carlosmiei We were already accepting `marketType = 'margin'`, so we're not removing it because it would break existing code. It would make no sense to add a new mode to read margin that is incomplete and faulty